### DISCUSSION TITLE
how to set  "save output to path:" to the same directory as the source file?

### DISCUSSION THREAD

**Discussion by kulkatt** [mifi/lossless-cut#1955] [Q&A] [answered]
I already manually selected a path.
![image](https://github.com/mifi/lossless-cut/assets/24421888/2b289514-5a7c-456b-8434-01f660adb9e2)

**Comment by mifi** [comment]
click the X next to the working dir button in the to right:

<img width="123" alt="Screenshot 2024-04-09 at 13 47 36" src="https://github.com/mifi/lossless-cut/assets/402547/f50861b2-f727-4e15-84ae-44003d075751">

**Reply by kulkatt** [reply]
thanks for the reply,
but I can't find the x either in settings or in export settings.
![image](https://github.com/mifi/lossless-cut/assets/24421888/30925479-28b6-4600-bd69-6c70210a9591)

**Reply by mifi** [reply]
it's not under settings. it's in the main player view

**Reply by kulkatt** [reply, marked as answer]
thanks, just found it
![image](https://github.com/mifi/lossless-cut/assets/24421888/7ec3382c-5f2f-4d9e-b48c-9db53cab2726)

**Reply by Grooves1210** [reply]
Please mark completed thread as "answered"
